lib/license_finder/platform.rb

```diff
@@ -1,11 +1,11 @@
 module LicenseFinder
   module Platform
     def self.darwin?
-      RUBY_PLATFORM =~ /darwin/
+      !(RUBY_PLATFORM =~ /darwin/).nil?
     end
 
     def self.windows?
-      RUBY_PLATFORM =~ /mswin|cygwin|mingw/
+      !(RUBY_PLATFORM =~ /mswin|cygwin|mingw/).nil?
```

Comment:
I had used double-negation myself in other Ruby projects, but then came across a project that uses RuboCop for the style checks and instructs not to use double-negation for style and readability reasons.